pyo3-benches/benches/bench_call.rs

```diff
@@ -33,9 +33,9 @@ fn bench_call_1(b: &mut Bencher<'_>) {
 
         let foo_module = &module.getattr("foo").unwrap();
         let args = (
-            <_ as IntoPy<PyObject>>::into_py(1, py).into_bound(py),
-            <_ as IntoPy<PyObject>>::into_py("s", py).into_bound(py),
-            <_ as IntoPy<PyObject>>::into_py(1.23, py).into_bound(py),
+            <_ as IntoPyObject>::into_pyobject(1, py).unwrap(),
+            <_ as IntoPyObject>::into_pyobject("s", py).unwrap(),
+            <_ as IntoPyObject>::into_pyobject(1.23, py).unwrap(),
```

Comment:
I think the path-based call might have been to avoid type inference problems with `"s".into_py(py)`, might not need any more here and below?

```suggestion
            1.into_pyobject(py).unwrap(),
            "s".into_pyobject(py).unwrap(),
            1.23.into_pyobject(py).unwrap(),
```